x/liquidstakeibc/keeper/host_chain.go

```diff
@@ -88,6 +88,31 @@ func (k *Keeper) ProcessHostChainValidatorUpdates(
 		k.SetHostChainValidator(ctx, hc, val)
 	}
 
+	// process LSM cap updates
+	if hc.Flags.Lsm {
+		// check if the validator has reached the LSM validator bond
+		var validatorHasRoomForDelegations bool
+		if validator.DelegatorShares.IsZero() {
+			validatorHasRoomForDelegations = true // if no shares are issued yet, the validator can accept more delegations
```

Comment:
are we sure, if validator has no delegator shares we can stake? What if there are no bonded tokens as well?